21/View/PokemonCell.swift

```diff
@@ -10,7 +10,7 @@ import Kingfisher
 
 class PokemonCell: UICollectionViewCell {
     
-    var pokemon: PokemonObject! {
+    var pokemon: Pokemon! {
```

Comment:
This should be optional. If something goes wrong with setting the Pokemon you don't want the code to crash